qemu/tests/timedrift.py

```diff
@@ -138,7 +143,7 @@ def restore_cpu_affinity(prev_masks):
                 host_load_sessions.append(load_cmd)
                 # Set the CPU affinity of the load process
                 pid = load_cmd.get_pid()
-                set_cpu_affinity(pid, cpu_mask)
+                set_cpu_affinity(pid, cpu_mask << i)
```

Comment:
Could you help explain why `cpu_mask << i`, why not `(pid, i)` directly? assuming that you have 4 cpu, `taskset -p 16 $pid` will actually set the affinity mask to 6, is that what you want?

Reply:
@sitoliu Okay, the phenomenon I described only happens on fedora, RHEL will set the right value pass to it. But I still wonder why not `i` directly.

Reply:
Because `taskset -p ` need a bitmask to set a process to specific cpu, the lowest order bit corresponding to the first logical CPU and the highest order bit corresponding to the last logical  CPU, `i` is not the right mask.

Reply:
@sitoliu Got it, thanks for the explanation.